auth_saml/readme/CONFIGURE.rst

```diff
@@ -1,5 +1,16 @@
-To use this module, you need an IDP server, properly set up. Go through the
+To use this module, you need an IDP server, properly set up.
 
-#. Configure the module according to your IDP's instructions
+#. Configure the module according to your IdP’s instructions
    (Settings > Users & Companies > SAML Providers).
 #. Pre-create your users and set the SAML information against the user.
+
+By default, the module let users have both a password and SAML ids.
+To increase security, disable passwords by using the option in Settings.
+Note that the admin account can still have a password, even if the option is activated.
+Setting the option immediately remove all password from users with a configured SAML ids.
+
+If all the users have a SAML id in a single provider, you can set automatic redirection
+in the provider settings. It is still possible to access the login without redirection
```

Comment:
maybe worth to mention that the order (sequence) of the providers matters here.

Reply:
I added the information.